src/module.zig

```diff
@@ -239,6 +316,22 @@ pub fn Objects(comptime T: type) type {
             return unpacked;
         }
 
+        /// Returns true if the field has an `updated` bit set in internal, after setting the bit
+        /// back to false.
```

Comment:
It doesn't make sense to mention `internal` in documentation comments for the public API; internal should be.. well.. an internal implementation detail the user of the API shouldn't need to think or worry about :)